h2o-genmodel/src/main/java/hex/genmodel/algos/ensemble/StackedEnsembleMojoReader.java

```diff
@@ -17,6 +17,9 @@ protected void readParentModelData() throws IOException {
         int baseModelNum = readkv("base_models_num", 0);
         _model._baseModelNum = baseModelNum;
         _model._metaLearner = getModel((String) readkv("metalearner"));
+        _model._metaLearnerTransform = (String) readkv("metalearner_transform", "NONE");
+        if (!_model._metaLearnerTransform.equals("NONE") && !_model._metaLearnerTransform.equals("Logit"))
```

Comment:
this is a good thing to have but we also need to make sure such mojo couldn't even reach the code (by version checking)

Reply:
I bumped the minor version of StackedEnsemble MOJO both in the writer and in the reader. I assume the version checking is done by `hex.genmodel.ModelMojoReader#checkMaxSupportedMojoVersion` or am I missing something?

Reply:
you are right, nothing else needed